cmake/Codegen.cmake

```diff
@@ -140,6 +140,25 @@ if (INTERN_BUILD_ATEN_OPS)
   ENDFOREACH()
   list(APPEND ATen_CPU_SRCS ${cpu_kernel_cpp})
 
+  FILE(GLOB cpu_quantized_kernel_cpp_in "${CMAKE_CURRENT_LIST_DIR}/../aten/src/ATen/native/quantized/cpu/kernels/*.cpp")
+  FOREACH(i RANGE ${NUM_CPU_CAPABILITY_NAMES})
+    FOREACH(IMPL ${cpu_quantized_kernel_cpp_in})
```

Comment:
Instead of copy-paste can you combine `cpu_kernel_cpp_in` and `cpu_kernel_cpp_in` to iterate them together.

It is prote to error otherwise as you already (likely) adding files twice, but not cleaning `cpu_kernel_cpp` content between loops.